### ISSUE TITLE
Consider moving Coturn and Exim subcharts to dedicated repos

### ISSUE THREAD

**Issue by Routhinator** [dacruz21/matrix-chart#28]
I've been thinking about this as far as future development and contributions go and since these same two services are used for many things outside Matrix (Coturn is needed by people deploying RocketChat and MM as well) - Seperating them into dedicated charts and listing them as a dependency here may encourage more collaboration on these components.

**Comment by typokign** [comment]
Yeah, that's probably a good idea. I've only scratched the surface of Coturn configuration in this chart though, and am nowhere near familiar enough with Coturn that I would feel comfortable taking ownership of a chart and testing it.

That being said, if you or anyone wants to split either of those charts out of this repo and expand on them, I would absolutely pull them in as dependencies.

Guess I'll flag this "help wanted" in case any Coturn experts stumble across this repo :)

**Comment by Routhinator** [comment]
I am likely to look at splitting CoTURN out after I contribute some improvements to it as it is embedded in this chart. Once we've dealt with TLS and configuration I'll split it out and maintain a stable itegration of it with this chart. I've opened #33 to discuss some immediate improvements to discovery/loadbalancing/config
